### ISSUE TITLE
[SCREENREADER]: Add aria-controls to the calendar widget buttons to assist JAWS users

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#6102]
## Description

I'd like to add some `aria-controls` attributes to our calendar widget buttons that expand and collapse the content. By adding these and unique IDs to each `<div>` JAWS users can press `JAWS + Alt + m` to move to the controlled element. It'll help improve the situation highlighted and not completely solved for #2709 . Code snippet and screen shot attached below.

## Related Issues
* https://github.com/department-of-veterans-affairs/va.gov-team/issues/2709
* https://github.com/department-of-veterans-affairs/va.gov-team/issues/6100

## Point of Contact

**VFS Point of Contact:** _Trevor_

## Acceptance Criteria

- [ ] JAWS users can press `JAWS + Alt + m` to move to the controlled element. There should be no "Failed to move to controlled element" errors.
- [ ] The visual display and functionality are not disrupted by the code adjustments
- [ ] The calendar widget (and page) does not throw any axe scan errors

## Environment

* Windows 10
* IE11, Chrome
* JAWS
* calendar widgets (direct booking, and AM/PM checkbox) views

## Possible Fixes (optional)

Adjusting the table container and adding a paragraph for SR-only caption text should work great:

```html
<!-- Add an aria-controls that references the nearest unique ID on divs -->
<button
  aria-controls="container-unique-1"
  aria-expanded="true"
  aria-label="Monday, February 24th"
  class="vaos-calendar__calendar-day-button"
  id="date-cell-2020-02-24"
  type="button"
>
  24
  <span class="vaos-calendar__day--selected-triangle"></span>
</button>
<!-- Add a unique ID to each div here -->
<div class="vaos-calendar__options-container" id="container-unique-1">
  <!-- The form elements go here -->
  ...
</div>
```

## WCAG or Vendor Guidance (optional)

* [Info and Relationships: Understanding SC 1.3.1](https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-programmatic.html)

## Screenshots or Trace Logs

![Screen Shot 2020-02-21 at 4 37 10 PM](https://user-images.githubusercontent.com/934879/75077996-03acba80-54ca-11ea-826e-34f467c36b97.png)

**Comment by jbalboni** [comment]
@1Copenut Does it matter that the div the button will be controlling doesn’t exist until that date is selected? It's rendered conditionally.

**Comment by 1Copenut** [comment]
@jbalboni Great question. It does matter for timing. The `aria-controls` attribute should be null or not exist until the ID is “there”. Otherwise it’ll thrown an axe error.
